### PR TITLE
Run prettier project

### PR DESCRIPTION
**Pull request by jonallured**
This PR updates prettier, runs it across the project and then adds the check command to the linting step so that CI will fail with malformed code.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit cafb68675adb56b8bb49300a1ca52d80bdf716ee

https://deploy-preview-2181--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx [hunk 1]
```diff
@@ -107,18 +107,16 @@ export class ArtworkSidebarBidAction extends React.Component<
...
-                Registration pending
-              </Button>
-            )}
-          {registrationAttempted &&
-            registeredToBid && (
-              <Button width="100%" size="large" mt={1} disabled>
-                Registration complete
-              </Button>
-            )}
+          {registrationAttempted && !registeredToBid && (
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx]: Interesting that this is a change -- it looks like prettier updated it's formatter style.

**Comment by damassi** [comment]
LGTM! 👍 

merge on green

**Comment by artsyit** [comment]
:rocket: PR was released in v15.5.9 :rocket:

**Pull request merged**